.github/workflows/tests.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
     strategy:
       matrix:
-        php: [7.3, 7.4, 8.0]
+        php: [7.3, 7.4, 8.0, 8.0]
```

Comment:
PHP 8.0 should not be added twice

Reply:
Yeah, little glitch there. We'll get that fixed up.